iceoryx_hoofs/include/iceoryx_hoofs/cxx/expected.hpp

```diff
@@ -234,6 +234,7 @@ class IOX_NO_DISCARD expected<ErrorType> : public FunctionalInterface<expected<E
     template <typename... Targs>
     static expected create_error(Targs&&... args) noexcept;
 
+    // AXIVION Next Construct AutosarC++19_03-A13.5.3: Implementation is inspired from std::expected
     /// @brief  returns true if the expected contains an error otherwise false
     /// @return bool which contains true if the expected contains an error
     explicit operator bool() const noexcept;
```

Comment:
@elfenpiff I'm wondering if this API is used in our codebase. I expected this to behave exactly the other way around, similar to the optional. Do you think we should keep this or make our users use the more explicit `has_error` method?

Reply:
@saif-at-github I talked to @elfenpiff and we can keep this but the documentation is wrong. Can you please replace `contains` with `does not contain` on line 238 and on line 354